pkg/kubelet/cm/devicemanager/endpoint_test.go

```diff
@@ -113,7 +113,7 @@ func TestAllocate(t *testing.T) {
 		{ID: "ADeviceId", Health: pluginapi.Healthy},
 	}
 	callbackCount := 0
-	callbackChan := make(chan int)
+	callbackChan := make(chan int, 1)
```

Comment:
could you please copy the rationale for your change in the commit message, and squash the two commits together?

Reply:
fixed! @fromanirh